code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/OptimizeExtension.java

```diff
@@ -61,11 +62,19 @@ public boolean doWork(final Event event) {
 
     // Map containing the update event IDs (and corresponding requested scopes) for Edge events that
     // haven't yet received an Edge completion response.
-    private Map<String, List<DecisionScope>> updateRequestEventIdsInProgress = new HashMap<>();
+    // Concurrent Map containing the update event IDs (and corresponding requested scopes) for Edge
+    // events that haven't yet received an Edge completion response.
+    // This is accessed from multiple threads.
```

Comment:
There are duplicates in this comment. Can you remove like 63 - 64

Reply:
removed @praveek